core/src/main/codegen/templates/Parser.jj

```diff
@@ -1312,20 +1313,26 @@ SqlSelect SqlSelect() :
         where = WhereOpt()
         groupBy = GroupByOpt()
         having = HavingOpt()
+<#list parser.qualifyClauseMethod as method>
+        qualify = ${method}
```

Comment:
nit: Should there be a \<#sep\> here?

Reply:
I'm not really sure how <#sep> could be used here. Could you clarify?

Reply:
Also maybe this could be clearer if qualifyClauseMethod was just a boolean and this was an if statement since there should only be one method inside this array

Reply:
Don't we usually add \<#sep\> | \</#sep\> when we inject a list of methods?

Reply:
I think that's only if each list item is an "or" statement. [<#sep>](https://freemarker.apache.org/docs/ref_directive_list.html#ref_list_sep) is used to display something between each item but not after the last item, but I we don't need "|" inserted here

Reply:
If there are multiple qualifyClauseMethod's, wouldn't we want a "|" between them? If there is just meant to be a single qualify parse method, maybe we should convert this to a #if instead?

Reply:
I think since there should only be one qualify parse method, it would be better to just make this an #if statement. Otherwise #list is kind of misleading in this case